nodes/clusters/nodes-cluster-resource-configure.adoc

```diff
@@ -26,6 +26,11 @@ managing application memory by:
 // assemblies.
 
 include::modules/nodes-cluster-resource-configure-about.adoc[leveloffset=+1]
+[discrete]
+[id="additional-resources_understanding-compute-resources"]
+== Additional resources
```

Comment:
Is this supposed to exist within "Understanding managing application memory"? If so, it needs to be an H3.

Reply:
Oh, whoops. Andrea's right. Our convention is to make it a  label. 

Not an appropriate a11y practice, but that's beyond the scope of your PR. :)